### ISSUE TITLE
get_random_word() & get_random_words() return a NoneType....!!

### ISSUE THREAD

**Issue by arixank** [vaibhavsingh97/random-word#70]
**Describe the bug**
At some point in type I'm encountering a NoneType in my code, which I'm unable to try and catch nor ignore it by another method..!

**To Reproduce**
here is a code screen shot:

![image](https://user-images.githubusercontent.com/50585400/121858689-7de2fd00-cd14-11eb-88c0-d05eacdc8a8b.png)

code:
``` from random_word import RandomWords
import random

rwd = RandomWords()
choice = random.randint(0, 14)

print(choice)
word = rwd.get_random_words(limit=15)
print(word, '\n')
new_word = word[choice]
print(new_word)
```

**Expected behavior**
Need to generate words or at least return a English word

**Screenshots**
Error Screen Shot
![image](https://user-images.githubusercontent.com/50585400/121858883-b2ef4f80-cd14-11eb-860f-b5575a3af737.png)

**Environment (please complete the following information):**
 - OS: [Linux]
 - Python [3.9]

**Comment by arixank** [comment]
Even This doesn't work:
![image](https://user-images.githubusercontent.com/50585400/121859835-d5ce3380-cd15-11eb-8acc-62934a6ae602.png)

**Comment by Maxwelldoug** [comment]
Same issue here under Ubuntu 20.04 LTS with python 3.9, I'll go try with 3.8

**Comment by arixank** [comment]
> Same issue here under Ubuntu 20.04 LTS with python 3.9, I'll go try with 3.8

yes please let me know, I'm also working on several work arounds..!!

Thanks

**Comment by arixank** [comment]
for now this is what I'm doing for a dictionary type application..! At least string type can be done with something.

![image](https://user-images.githubusercontent.com/50585400/121897359-ea272600-cd3f-11eb-8886-cc9ec9525221.png)

**Comment by romanmeister** [comment]
Same here, strangely enough it sometimes spits out a `list` type, sometimes `NoneType`
```python
r = RandomWords()
random_words_list = r.get_random_words(limit=length_of_list)
```

**Comment by Mohamed-Amine-AALI** [comment]
Same here

`from random_word import RandomWords
r = RandomWords()

print(r.get_random_word())`

Sometimes it prints an actual word, sometimes None

**Comment by kikefdezl** [comment]
Still happening for me:

```
import random
from random_word import RandomWords

random_words = RandomWords()
words = random_words.get_random_words(limit=random.randint(1, 10), hasDictionaryDef=True)
```

This sometimes returns a list of random words, and sometimes returns NoneType.

**Comment by Anas812001** [comment]
I managed to fix it guys
![image](https://user-images.githubusercontent.com/56180102/182770289-bc3b0b66-9a2c-4230-a0f9-9de49fcb22f6.png)

**Comment by Maxwelldoug** [comment]
That's not a fix, it's a hacky workaround at best.
